apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -154,6 +155,13 @@ $welcome-tour-button-background-color: #32373c; // former $dark-gray-700. TODO:
 	}
 }
 
+@include break-mobile {
+	.wpcom-editor-welcome-tour-frame .welcome-tour-card__overlay-controls .components-button,
+	.welcome-tour-card__overlay-controls-visible .components-button {
+		opacity: 0.9 !important;
```

Comment:
I'd prefer we avoid `!important` constraints on entities as they are too restrictive for anything else that may want to override.

Reply:
Earlier while testing I was trying hard why css is not getting applied. That's when I added `!important`. Later I released I cannot test plugin. So I think by mistake I left it there.